go.mod

```diff
@@ -3,9 +3,9 @@ module k8s.io/repo-infra
 go 1.13
 
 require (
-	github.com/bazelbuild/bazel-gazelle v0.20.0
+	github.com/bazelbuild/bazel-gazelle v0.21.1
 	github.com/bazelbuild/buildtools v0.0.0-20200228172928-c9d9e342afdb
-	github.com/golangci/golangci-lint v1.25.0
-	golang.org/x/build v0.0.0-20200302185339-bb8466fe872a
-	k8s.io/klog/v2 v2.0.0
+	github.com/golangci/golangci-lint v1.29.0
+	golang.org/x/build v0.0.0-20200720211405-d0191c8228ec
+	k8s.io/klog/v2 v2.3.0
```

Comment:
Can we move all these version bumps to their own PR?

Reply:
Yep! This was more so yak shaving in support of the [k/k go1.15 bump](https://github.com/kubernetes/kubernetes/pull/93264).

I'll split it out.

Reply:
Deps bump in https://github.com/kubernetes/repo-infra/pull/198.